testing/src/main/java/io/stargate/it/http/graphql/graphqlfirst/TtlTest.java

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright The Stargate Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.stargate.it.http.graphql.graphqlfirst;
+
+import com.datastax.oss.driver.api.core.CqlIdentifier;
+import com.datastax.oss.driver.api.core.CqlSession;
+import com.jayway.jsonpath.JsonPath;
+import io.stargate.it.driver.CqlSessionExtension;
+import io.stargate.it.driver.TestKeyspace;
+import io.stargate.it.http.RestUtils;
+import io.stargate.it.storage.StargateConnectionInfo;
+import org.assertj.core.api.InstanceOfAssertFactories;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.data.Offset.offset;
+import static org.junit.jupiter.params.provider.Arguments.arguments;
+
+@ExtendWith(CqlSessionExtension.class)
+public class TtlTest extends GraphqlFirstTestBase {
+
+  private static GraphqlFirstClient CLIENT;
+  private static String KEYSPACE;
+  private static UUID SCHEMA_VERSION;
+
+  @BeforeAll
+  public static void setup(StargateConnectionInfo cluster, @TestKeyspace CqlIdentifier keyspaceId) {
+    CLIENT =
+        new GraphqlFirstClient(
+            cluster.seedAddress(), RestUtils.getAuthToken(cluster.seedAddress()));
+    KEYSPACE = keyspaceId.asInternal();
+    SCHEMA_VERSION =
+        CLIENT.deploySchema(
+            KEYSPACE,
+            "type Foo @cql_input{\n"
+                + "  k: Int @cql_column(partitionKey:true)\n"
+                + "  v: Int\n"
+                + "}\n"
+                + "type Mutation {\n"
+                + "  insertFoo(foo: FooInput): Foo @cql_insert(ttl: \"3600\")\n"
+                + "  updateFoo(k: Int, v: Int): Boolean @cql_update(targetEntity: \"Foo\", ttl: \"PT2H\")\n"
```

Comment:
I went with static directive arguments. I think passing the TTL as an argument is not a likely use case. However that can always be added later if it gets requested.